.github/actions/prettier/action.yml

```diff
@@ -0,0 +1,29 @@
+# Define action name, description, and inputs
+name: "Prettier Lint"
+description: "Lint code with Prettier"
```

Comment:
_nit_ - we're using both Prettier and ESLint here, but action name and directory name suggest otherwise :)